packages/cli-e2e/utils/cli.ts

```diff
@@ -48,13 +50,30 @@ export function answerPrompt(
   });
 }
 
+export interface ISetupUIProjectOptionsArgs {
+  flags?: string[];
+  timeout?: number;
+}
+
 export async function setupUIProject(
-  commandArgs: string,
+  uiCommand: string,
   projectName: string,
-  cliProcesses: ChildProcessWithoutNullStreams[]
+  cliProcesses: ChildProcessWithoutNullStreams[],
+  options: ISetupUIProjectOptionsArgs = {}
 ) {
+  const uiProjectFolderName = 'ui-projects';
+  ensureDirSync(uiProjectFolderName);
```

Comment:
Where does that ends up?
It should be added to the ignore list of prettier, eslint, git, npm and jest